Incorporate documentation feedback from Ceph - Mon migration procedure (#729)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_migrating-mon-from-controller-nodes-config-target-nodes.adoc

$ sudo systemctl restart iptables
----

. If `nftables` is used in the existing deployment, edit `/etc/nftables/tripleo-rules.nft`

> and add the following lines:

Reviewer: ```suggestion
and add the following content:
```

Author: Done

+
----
# 110 ceph_mon {'dport': [6789, 3300, '9100']}
add rule inet filter TRIPLEO_INPUT tcp dport { 6789,3300,9100 } ct state new counter accept comment "110 ceph_mon"
----

> . Save the file.

Reviewer: ```suggestion
. Save the file.

```

Author: Done

. Restart the `nftables` service:
+
----
$ sudo systemctl restart nftables
----

> . Verify the rules have been applied:

Reviewer: ```suggestion
. Verify that the rules are applied:
```

Author: Done

----
+
----
$ sudo cephadm shell -- ceph orch host ls
HOST              ADDR           LABELS          STATUS
ceph-0        192.168.24.14  osd mon mgr _admin
ceph-1        192.168.24.7   osd mon mgr _admin
ceph-2        192.168.24.8   osd mon mgr _admin
controller-0  192.168.24.15  _admin mgr mon
controller-1  192.168.24.23  _admin mgr mon
controller-2  192.168.24.13  _admin mgr mon
----

Setup a Ceph client on the first controller node that is used during the rest
of the procedure to interact with Ceph.

Setup an additional IP address on the storage network that is used to

> interact with Ceph when the first controller node is decommissioned.

Reviewer: Are these two sentences supposed to be individual steps?

Author: mm no, just the same explanation of the steps below, I can make them without the extra line in the middle if the wording is good enough for you.

> . Backup the content of `/etc/ceph` in the `ceph_client_backup` directory.

Reviewer: ```suggestion
. Back up the content of `/etc/ceph` in the `ceph_client_backup` directory.
```

Author: Done

+
----
$ mkdir -p $HOME/ceph_client_backup
$ sudo cp -R /etc/ceph/* $HOME/ceph_client_backup
----

> . Edit `/etc/os-net-config/config.yaml` and add `- ip_netmask: 172.17.3.200`
>   after the ip address on the vlan that belongs to the storage network. Replace
>   `172.17.3.200` with any other available ip address on the storage network
>   that can be statically assigned to controller-0.

Reviewer: ```suggestion
. Edit `/etc/os-net-config/config.yaml` and add `- ip_netmask: 172.17.3.200`
  after the IP address on the VLAN that belongs to the storage network. Replace
  `172.17.3.200` with any other available IP address on the storage network
  that can be statically assigned to `controller-0`.
```

Author: Done

> . Save the file and refresh controller-0 network configuration:

Reviewer: ```suggestion
. Save the file and refresh the `controller-0` network configuration:
```

Author: Done

+
----
$ sudo os-net-config -c /etc/os-net-config/config.yaml
----

> . Verify that the IP address is present in the controller node.

Reviewer: ```suggestion
. Verify that the IP address is present in the Controller node:
```

Author: Done

+
----
$ ip -o a | grep 172.17.3.200
----

> . Ping the ip address and confirm that it is reachable:

Reviewer: ```suggestion
. Ping the IP address and confirm that it is reachable:
```

Author: Done

+
----
$ ping -c 3 172.17.3.200
----

> . Verify that you can interact with the ceph cluster:

Reviewer: ```suggestion
. Verify that you can interact with the {Ceph} cluster:
```

Author: Done

----
+
----
$ sudo cephadm shell -- ceph orch host ls
HOST              ADDR           LABELS          STATUS
ceph-0        192.168.24.14  osd mon mgr _admin
ceph-1        192.168.24.7   osd mon mgr _admin
ceph-2        192.168.24.8   osd mon mgr _admin
controller-0  192.168.24.15  _admin mgr mon
controller-1  192.168.24.23  _admin mgr mon
controller-2  192.168.24.13  _admin mgr mon
----

> Setup a Ceph client on the first controller node that is used during the rest
> of the procedure to interact with Ceph. Setup an additional IP address on the
> storage network that is used to interact with Ceph when the first controller
> node is decommissioned.

Reviewer: ```suggestion
. Set up a Ceph client on the first Controller node that is used during the rest
of the procedure to interact with {Ceph}. Set up an additional IP address on the
storage network that is used to interact with {Ceph} when the first Controller
node is decommissioned:
```

Reviewer: @fmount I formatted this text as a step and the rest as substeps. The reason is because this text was interfering with the order of the steps. The steps' order started over after this text.

> . Back up the content of `/etc/ceph` in the `ceph_client_backup` directory.

Reviewer: ```suggestion
.. Back up the content of `/etc/ceph` in the `ceph_client_backup` directory.
```

Author: Done

+
----
$ mkdir -p $HOME/ceph_client_backup
$ sudo cp -R /etc/ceph/* $HOME/ceph_client_backup
----

> . Edit `/etc/os-net-config/config.yaml` and add `- ip_netmask: 172.17.3.200`

Reviewer: ```suggestion
.. Edit `/etc/os-net-config/config.yaml` and add `- ip_netmask: 172.17.3.200`
```

Author: Done

  after the IP address on the VLAN that belongs to the storage network. Replace
  `172.17.3.200` with any other available IP address on the storage network
  that can be statically assigned to `controller-0`.

> . Save the file and refresh the `controller-0` network configuration:

Reviewer: ```suggestion
.. Save the file and refresh the `controller-0` network configuration:
```

+
----
$ sudo os-net-config -c /etc/os-net-config/config.yaml
----

> . Verify that the IP address is present in the Controller node:

Author: Done

+
----
$ ip -o a | grep 172.17.3.200
----

> . Ping the IP address and confirm that it is reachable:

Reviewer: ```suggestion
.. Ping the IP address and confirm that it is reachable:
```

Author: Done

+
----
$ ping -c 3 172.17.3.200
----

> . Verify that you can interact with the {Ceph} cluster:

Reviewer: ```suggestion
.. Verify that you can interact with the {Ceph} cluster:
```

Author: Done
